src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int bottomBase;
+    private int topBase;
+    private int hight;
+
+    public IsoscelesTrapezoid(int bottomBase, int topBase, int hight, Color color) {
+        this.bottomBase = bottomBase;
+        this.topBase = topBase;
+        this.hight = hight;

```

Comment:
Ensure that the parameter name `hight` is corrected to `height` in the constructor as well.